src/main/java/com/flagsmith/FlagsmithConfig.java

```diff
@@ -55,7 +57,7 @@ protected FlagsmithConfig(Builder builder) {
         private int readTimeoutMillis = DEFAULT_READ_TIMEOUT_MILLIS;
         private SSLSocketFactory sslSocketFactory;
         private X509TrustManager trustManager;
-        private Interceptor interceptor;
+        private List<Interceptor> interceptor = new ArrayList<>();
```

Comment:
Could we change the attribute name to `interceptors` to indicate that it's an array?

Reply:
done